html/MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include avocado_result_html/resources *
```

Comment:
I'd propose something simpler at this point:

```diff
diff --git a/Makefile b/Makefile
index fcd15c3..fef6d9b 100644
--- a/Makefile
+++ b/Makefile
@@ -168,6 +168,7 @@ modules_boundaries:
 
 develop:
 	$(PYTHON) setup.py develop $(shell $(PYTHON26) || echo --user)
+   $(PYTHON) html/setup.py develop $(shell $(PYTHON26) || echo --user)
 
 link: develop
 	for MAKEFILE in $(AVOCADO_PLUGINS);\

```

Nothing prevents us from expanding on your idea, but I think we don't need anything more complex than this at this point.

PS: I honestly think we're using too much of `Makefile` in inspite of support already in `setup.py`, setuptools, pip, etc.  I'm going go send a few proposals to get more inline with the more standard Pythonic way of doing some of those things.